docs/index.html

```diff
@@ -6,25 +6,25 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
 	<title>Command Line Tools – Algorithmia</title>
-	<link rel="canonical" href="http://cli.algorithmia.com/" />
+	<link rel="canonical" href="https://algorithmia.com/developers/clients/cli" />
 	<meta name="description" content="Command line tools for using and authoring Algorithmia algorithms.">
 	<meta name="robots" content="index,follow">
 
 	<!-- FB -->
 	<meta property="og:title" content="Algorithmia Command Line Tools">
-	<meta property="og:url" content="http://cli.algorithmia.com/">
+	<meta property="og:url" content="https://algorithmia.com/developers/clients/cli">
 	<meta property="og:description" content="Command line tools for using and authoring Algorithmia algorithms">
 	<meta property="og:locale" content="en_US" />
 	<meta property="og:type" content="website" />
-	<meta property="og:image" content="http://cli.algorithmia.com/public/images/hero-fb.png" />
+	<meta property="og:image" content="https://algorithmia.com/developers/clients/cli" />
 	<meta property="og:site_name" content="Algorithmia" />
 
 	<!-- TWTR -->
 	<meta name="twitter:card" content="summary_large_image" />
 	<meta name="twitter:site" content="@algorithmia" />
 	<meta name="twitter:title" content="Algorithmia Command Line Tools" />
 	<meta name="twitter:description" content="Command line tools for using and authoring Algorithmia algorithms" />
-	<meta name="twitter:image" content="http://cli.algorithmia.com/public/images/hero-twttr.png" />
+	<meta name="twitter:image" content="https://algorithmia.com/developers/clients/cli" />
```

Comment:
This should be an image. Let's just reuse what the algorithmia landing page is using:
```suggestion
	<meta name="twitter:image" content="https://cdn.algorithmia.com/static/images/algo-share-image.f20f48e.png" />
```